status-im/status-react
# Allow easy status-go log level modification via feature flag (#2756)

Fixes #2756 

### Summary:

The log level passed by the Android and iOS code to status-go was hard-coded to `"INFO"`. This PR uses the log level defined by a feature flag in the `.env` files allowing status-go to get the correct value to initialize a node, taking the correct case conversion into consideration. No change was required on the Go side because the log level is already being read from the node config JSON: https://github.com/status-im/status-go/blob/develop/geth/node/manager.go#L498.
In the process, got rid of superfluous `DEBUG_LOGS_ENABLED` feature flag using the value of `LOG_LEVEL` to keep functionality identical.
...

### Steps to test:
- Open debug build of Status
- Verify that the logs contain `debug` level entries

status: ready
